src/submission/forms.py

```diff
@@ -316,8 +318,7 @@ class ConfiguratorForm(forms.ModelForm):
 
     def __init__(self, *args, **kwargs):
         super(ConfiguratorForm, self).__init__(*args, **kwargs)
-        self.fields[
-            'default_section'].queryset = models.Section.objects.filter(
+        self.fields['default_section'].queryset = models.Section.objects.language().fallbacks('en').filter(
```

Comment:
should be settings.LANGUAGE_CODE instead of  `'en'`

Reply:
Roger that.